extensions/src/main/java/org/pentaho/platform/plugin/services/email/EmailConfigurationXml.java

```diff
@@ -66,6 +68,15 @@
   private static final String DEFAULT_EMAIL_CONFIG_PATH = "system" + File.separator + "smtp-email" + File.separator
       + "email_config.xml";
 
+  private static Encr encr;
+  static {
+    try {
+      encr = Encr.getInstance();
+    } catch ( PasswordEncoderException | org.pentaho.support.utils.XmlParseException e ) {
+      //should never happen.  If it was going to throw an exception, it would have done so on boot
```

Comment:
Can you rethrow as a runtime exception, or log?  
I've seen enough cases where the "should never happen" expectation is proven wrong.

Also, since this is static, we really have no way of knowing exactly when this will get hit, right?  Could be earlier or later in the boot process if the jvm needs to load the class for any reason.